src/hooks/useCountryImage.js

```diff
@@ -1,30 +1,28 @@
 import { useState, useEffect } from 'react';
-import Countries from '../utility/countries';
 
-export const useCountryImage = (countryCode) => {
+export const useCountryImage = (countryName) => {
   const [isLoading, setLoading] = useState(true);
   const [error, setError] = useState('');
   const [countryImage, setCountryImage] = useState(null);
 
-  const country = Countries.getByAlphaCode(countryCode);
-  const randomImage = Math.floor(Math.random() * 10);
   const API =
     'https://api.unsplash.com/search/photos?client_id=rTLPnTqzq-rK3TnatKGFvmVeFGaZeYToaKUyzwcJOiA&query=';
 
-  const getImage = async () => {
-    try {
-      const response = await fetch(`${API}${country.name}`);
-      const data = await response.json();
-      setCountryImage(data.results[randomImage].urls.regular);
-    } catch (error) {
-      setError(error.message);
-    }
-    setLoading(false);
-  };
-
   useEffect(() => {
-    getImage();
-  }, [countryCode]);
+    // https://es.reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies
+    const randomImage = Math.floor(Math.random() * 10);
+    const getImage = async (name) => {
```

Comment:
@wolftrax5 I got a question, what is the difference between declaring this function inside or outside the useEffect hook? Thanks